app/src/main/java/com/zeapo/pwdstore/autofill/oreo/AutofillPreferences.kt

```diff
@@ -25,8 +25,18 @@ enum class DirectoryStructure(val value: String) {
     }
 
     fun getIdentifierFor(file: File) = when (this) {
-        FileBased -> file.parentFile?.name
-        DirectoryBased -> file.parentFile?.parentFile?.name
+        FileBased -> file.parentFile.name
+        DirectoryBased -> file.parentFile.parentFile?.name
+    }
+
+    fun getPathToIdentifierFor(file: File) = when (this) {
```

Comment:
I can't infer what this function is doing purely from the name. If we can't come up with a more explanatory name, please add a KDoc block explaining it.

Reply:
Added a KDoc since I couldn't think of a better name. Still open to rename this though.